src/components/Checklist/Checklist.tsx

```diff
@@ -1,75 +1,71 @@
-                </span>
-              );
-            })}
-          </IonContent>
-        </IonPopover>
+    <div>
+      <div className={styles.info}>
+        <Markdown>{fieldworkChecklist.info}</Markdown>
       </div>
-
       {/* Render each section in its own accordion. */}
-      {soilPackageChecklist.sections.map((s) => (
-        <IonAccordion key={s.title}>
-          <IonItem slot={"header"}>
-            <IonLabel>{s.title}</IonLabel>
-          </IonItem>
-          <div className={"ion-padding"} slot={"content"}>
+      {fieldworkChecklist.sections.map((s) => (
+        <div key={s.title}>
+          <SectionHeader>{s.title}</SectionHeader>
+          <div slot={"content"}>
+            <Markdown className={styles.info}>{s.info}</Markdown>
             <Markdown
               // Map Markdown elements to React elements.
               components={{
                 ul: (props) => <IonList>{props!.children}</IonList>,
-                li: (props) => (
-                  <IonItem className={styles.listItem} lines={"none"}>
-                    <IonIcon
-                      size={"small"}
-                      slot={"start"}
-                      icon={checkmarkCircle}
-                      color={"primary"}
-                    />
-                    <span>{props!.children}</span>
-                  </IonItem>
-                ),
+                li: (props) => {
+                  // use different icon for the nested list item
+                  const match = /^NESTEDLISTITEM/.exec(
+                    props!.children?.toString() || "",
+                  );
+                  return match ? (
+                    <IonItem className={styles.listItem} lines={"none"}>
+                      <IonIcon
+                        className={styles.listIcon}
+                        size={"small"}
+                        slot={"start"}
+                        icon={checkmarkCircleOutline}
+                        color={"primary"}
+                      />
+                      <IonLabel className={styles.listLabel}>
+                        {props!.children
+                          ?.toString()
+                          .replace(/NESTEDLISTITEM /, "")}
+                      </IonLabel>
+                    </IonItem>
+                  ) : (
+                    <IonItem className={styles.listItem} lines={"none"}>
+                      <IonIcon
+                        className={styles.listIcon}
+                        size={"small"}
+                        slot={"start"}
+                        icon={checkmarkCircle}
+                        color={"primary"}
+                      />
+                      <IonLabel className={styles.listLabel}>
+                        {props!.children}
+                      </IonLabel>
+                    </IonItem>
+                  );
+                },
               }}
             >
               {s.md}
             </Markdown>
+            <br></br>
```

Comment:
I assume these were added so that there is some extra vertical margin below the rendered Markdown. I would prefer this be achieved using CSS instead of line break HTML elements.

Reply:
Want to add extra vertical margin at the end of the content. So, will remove the br from the section loop and add it after the loop.
![Screenshot 2024-08-15 at 10 07 59 AM](https://github.com/user-attachments/assets/0b286138-4fdc-4779-af5b-64baab269ffd)
![Screenshot 2024-08-15 at 10 08 27 AM](https://github.com/user-attachments/assets/8a4f6599-d6fe-458a-8476-bb30d5c4089b)

Reply:
Ionic provides utility CSS classes that could be useful here: https://ionicframework.com/docs/layout/css-utilities#content-space